updatecli.d/helm-reporting-k8s-templates.yaml

```diff
@@ -7,7 +7,7 @@ sources:
       name: reporting-k8s-templates
       versionfilter:
         kind: semver
-        pattern: ^1.1.12
+        pattern: ^1.1.14
```

Comment:
This kind of change is usually not required. If you want to trigger a version update, just trigger the `Update image tags and Helm versions` action.